dali/operators/reader/loader/video/frames_decoder.cc

```diff
@@ -257,10 +259,13 @@ bool FramesDecoder::ReadFlushFrame(uint8_t *data, bool copy_to_output) {
 
   if (copy_to_output) {
     CopyToOutput(data);
-    LOG_LINE << "Read frame (ReadFlushFrame), timestamp " << av_state_->frame_->pts << std::endl;
   }
 
+  LOG_LINE << "Read frame (ReadFlushFrame), index " << next_frame_idx_ << " timestamp " <<
+    std::setw(5) << av_state_->frame_->pts << ", current copy " << copy_to_output << std::endl;
   ++next_frame_idx_;
+
+  // TODO(awolant): Figure out how to handle this during index building
```

Comment:
Something I discovered during this task. Will be handled in the future, as this is minor inconvenience.

Reply:
So how this is handled now?

Reply:
It was always handled through checking the return value of the decoding function.
The problem right now is that during the index build, we don't yet have number of frames. I used that to set the index of the current frame. So during the index building, index of the last frame is not set correctly. This shows in log.
After the index build we immediately call `Reset` and it sets the current frame index to 0, so everything is fine. For now, I wanted to left it as it is, because all of the ideas I have atm involve web of ifs or something like that. And the only consequence is that in the log for the index building we have index of the last frame -1 instead of whatever it should be. I'll figure this out later.